.storybook/env.js

```diff
@@ -1,3 +1,3 @@
-export const CLIENTID = 'ac77046c-156c-40f0-8507-3b5a58034582';
+export const CLIENTID = '8c5e1ebf-9177-4a91-afb6-ee63fb7b2aef';
```

Comment:
Where is this Client Id from?

Reply:
Now coming from the central managed App ID.